resolwe/storage/connectors/transfer.py

```diff
@@ -146,9 +146,13 @@ def transfer_objects(
 
         return None if objects_stored is objects else objects_stored
 
-    def _transfer_chunk(self, url: Path, objects: Iterable[dict]) -> bool:
+    def transfer_chunk(self, url: Path, objects: Iterable[dict]) -> bool:
```

Comment:
Commit message here is a bit misleading, as the commit only allows overriding of variables and has nothing to do with parallelization of upload (at least not directly).